### PR TITLE
fix: Company fields replaced with Organisation

### LINKED ISSUE #5854: Speaker and Attendee Forms: Change "Company" field to "Organisation"
In the Speaker and Attendee Forms please change "Company" field to "Organisation". Please ensure this shows up in the same way:
* on the wizard
* on the public speaker form
* on the attendee form during the order process

### PR DESCRIPTION
**Pull request by Saumyachitransh01**
Fixes #5854 

#### Short description of what this resolves:

Changed "Company" field to "Organisation" in Speaker and Attendee Form

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by daretobedifferent18 — DISMISSED** [review]
Working fine for me

**Review by maze-runnar — APPROVED** [review]

**Comment by mariobehling** [comment]
Existing events are still showing "company" in attendee form. Please resolve. Thank you.

Screenshot from 2020-12-10 07-07-12

**Comment by Saumyachitransh01** [comment]
Done
Screenshot from 2020-12-10 12-12-23

**Review by daretobedifferent18 — CHANGES_REQUESTED** [review]
Pls change in validation rules ```company``` to ```organisation```
for eg. ```'Please enter your company'``` -> ```'Please enter your organisation'```

File: app/mixins/custom-form.js [hunk 1]
```diff
@@ -380,7 +380,7 @@ export default Mixin.create(MutableArray, {
         event           : parent
       }),
       this.store.createRecord('custom-form', {
-        fieldIdentifier : 'company',
+        fieldIdentifier : 'organisation',
```
Comment (iamareebjamal) [inline app/mixins/custom-form.js:383]: Won't work. There is no field in server named organisation

File: app/models/attendee.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ export default ModelBase.extend({
   email              : attr('string'),
   jobTitle           : attr('string'),
   phone              : attr('string'),
-  company            : attr('string'),
+  organisation       : attr('string'),
```
Comment (iamareebjamal) [inline app/models/attendee.js:22]: Won't work

File: app/models/custom-form.js [hunk 3]
```diff
@@ -80,7 +80,7 @@ export default ModelBase.extend({
     billingAddress  : 'Billing Address',
     homeAddress     : 'Home Address',
     shippingAddress : 'Shipping Address',
-    company         : 'Company',
+    organisation    : 'Organisation',
```
Comment (iamareebjamal) [inline app/models/custom-form.js:83]: Won't work

File: app/models/user.js [hunk 4]
```diff
@@ -54,7 +54,7 @@ export default ModelBase.extend({
   billingContactName    : attr('string'),
   billingPhone          : attr('string'),
   billingCountry        : attr('string'),
-  company               : attr('string'),
+  organisation          : attr('string'),
```
Comment (iamareebjamal) [inline app/models/user.js:57]: Won't work

File: app/mixins/custom-form.js (hunk 1 above)
Comment (daretobedifferent18) [inline app/mixins/custom-form.js:383]: @Saumyachitransh01 I have just asked you to changes messages places. Pls revert all the places where not required you did in last commit

**Comment by Saumyachitransh01** [comment]
This PR will implement for newly created events from now onwards. But for existing events. It needs server side migration.

**Comment by maze-runnar** [comment]
@iamareebjamal for what the migration is actually needed in this PR? for alter_column?

**Comment by iamareebjamal** [comment]
No, just change the name of custom form field of company to organizer.

**Comment by iamareebjamal** [comment]
Change this in the open PR for custom form migration only

File: app/components/forms/user-payment-info-form.js [hunk 5]
```diff
@@ -10,7 +10,7 @@ import { countries } from 'open-event-frontend/utils/dictionary/demography';
 export default class UserPaymentInfoForm extends Component.extend(FormMixin) {
   didInsertElement() {
     super.didInsertElement(...arguments);
-    this.set('userBillingInfo', pick(this.authManager.currentUser, ['billingContactName', 'billingCity', 'billingPhone', 'company', 'billingTaxInfo', 'billingCountry', 'billingState', 'billingAddress', 'billingZipCode', 'billingAdditionalInfo']));
+    this.set('userBillingInfo', pick(this.authManager.currentUser, ['billingContactName', 'billingCity', 'billingPhone', 'organisation', 'billingTaxInfo', 'billingCountry', 'billingState', 'billingAddress', 'billingZipCode', 'billingAdditionalInfo']));
```
Comment (iamareebjamal) [inline app/components/forms/user-payment-info-form.js:13]:
```suggestion
    this.set('userBillingInfo', pick(this.authManager.currentUser, ['billingContactName', 'billingCity', 'billingPhone', 'company', 'billingTaxInfo', 'billingCountry', 'billingState', 'billingAddress', 'billingZipCode', 'billingAdditionalInfo']));
```

File: app/components/forms/user-payment-info-form.js [hunk 6]
```diff
@@ -29,12 +29,12 @@ export default class UserPaymentInfoForm extends Component.extend(FormMixin) {
             }
           ]
         },
-        company: {
-          identifier : 'company',
+        organisation: {
+          identifier : 'organisation',
```
Comment (iamareebjamal) [inline app/components/forms/user-payment-info-form.js:33]:
```suggestion
        company: {
          identifier : 'company',
```

**Pull request merged**